### ISSUE TITLE
Main Builds fail on first test fail

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext#2175]
Builds on main and branches seem to fail on first test failure
is this intentional. in the past we ran all tests
and then let jenkins report all fails

cc @LorenzoBettini

**Comment by LorenzoBettini** [comment]
> Builds on main and branches seem to fail on first test failure is this intentional. in the past we ran all tests and then let jenkins report all fails
> 
> cc @LorenzoBettini

yes, I changed that because I find it better to fail immediately than to wait for a few hours and then inspect the log to spot the problem.

However, if you prefer the old way, I'm OK.

**Comment by cdietrich** [comment]
@szarnekow what do you think

also: should we mark all flaky test with the flaky rule and or ignore them?

**Comment by LorenzoBettini** [comment]
We could also try this: https://tycho.eclipseprojects.io/doc/latest/tycho-surefire-plugin/integration-test-mojo.html#rerunFailingTestsCount
